azure-quantum/azure/quantum/optimization/solvers.py

```diff
@@ -248,25 +288,60 @@ def check_set_positive_int(self, var: int, var_name: str):
             self.set_one_param(var_name, var)
 
     def check_set_float_limit(
-        self, var: float, var_name: str, var_limit: float
+        self, var: float, var_name: str,
+        lower_bound_exclusive: Optional[float] = None,
+        lower_bound_inclusive: Optional[float] = None,
     ):
-        """Check whether the var parameter is a float larger than var_limit.
+        """Check whether the var parameter is a float satisfying bounds.
```

Comment:
What is "var"? Is this "variable" or "variance"? And if it's the former could we come up with a more descriptive name?

Reply:
This is the original name of the argument; I did not introduce it.
It seems to be used as a synonym for parameter in this code.
I changed all occurrences to `parameter_value` and `parameter_name` instead.

In my opinion, this is not the type of cleanup to request in a feature PR, it belongs in its own PR.

Reply:
@andrist thanks for doing that, I agree this would have been fine to do in a separate PR; I wasn't use if you had introduced that argument or not. The effort is still much appreciated!